File: 250719/일의 자리 배열/array-with-units.py
# 입력
arrayInt = list(map(int, input().split()))
arrayResult = []

# 반복문
for i in range(8) :
    intSum = arrayInt[i] + arrayInt[i + 1]
    if intSum > 9 : # 두 수의 합이 10 이상일 경우
        intSum -= 10
    arrayInt.append(intSum)

# 출력
for i in range(len(arrayInt)) :
    print(arrayInt[i], end = " ")

File: 250719/일의 자리 배열/array-with-units.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int[] arrayInt = new int[10];
        for (int i = 0; i < 2; i++) {
            arrayInt[i] = sc.nextInt();
        }

        // 반복문
        for (int i = 0; i < 8; i++) {
            int intSum = arrayInt[i] + arrayInt[i+1];
            if (intSum > 9) {
                intSum -= 10;
            }
            arrayInt[i+2] = intSum;
        }

        for (int i = 0; i < arrayInt.length; i++) {
            System.out.print(arrayInt[i] + " ");
        }
    }
}